pkg/interfaces/contracts/liquidity-mining/IVotingEscrowRemapper.sol

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General external License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General external License for more details.
+
+// You should have received a copy of the GNU General external License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+import "./IOmniVotingEscrow.sol";
+import "./IVotingEscrow.sol";
+
+/**
+ * @notice This contract allows veBAL holders on Ethereum to assign their balance to designated addresses on each L2.
+ * This is intended for smart contracts that are not deployed to the same address on all networks. EOA's are
+ * expected to either use the same address, or manage delegation on L2 networks themselves.
+ *
+ * @dev For each network (chainId), we maintain a mapping between local (Ethereum) and remote (L2) addresses.
+ * This contract remaps balance queries on remote network addresses to their corresponding local addresses.
+ * Users able to call this contract can set their own mappings, or delegate this function to another account if they
+ * cannot.
+ */
+interface IVotingEscrowRemapper {
+    event OmniVotingEscrowUpdated(IOmniVotingEscrow indexed newOmniVotingEscrow);
+    event AddressMappingUpdated(address indexed localUser, address indexed remoteUser, uint256 indexed chainId);
+    event RemoteAddressMappingCleared(address indexed remoteUser, uint256 indexed chainId);
+    event AddressDelegateUpdated(address indexed localUser, address indexed delegate);
+
+    /**
+     * @notice Returns Voting Escrow contract address.
+     */
+    function getVotingEscrow() external view returns (IVotingEscrow);
+
+    /**
+     * @notice Returns Omni Voting Escrow contract address.
+     */
+    function getOmniVotingEscrow() external view returns (IOmniVotingEscrow);
+
+    /**
+     * @notice Returns the current total supply of veBAL as a Point.
+     * @dev We return the total supply as a Point to allow extrapolating this into the future. Note that this
```

Comment:
Maybe link to IVotingEscrow where Point is defined? Though I guess the interface name is already given in the return value, so maybe that's good enough.